### PR TITLE
Add simple test v4

### PR DESCRIPTION
**Pull request by bavery22**
addressed comments.

### CODE REVIEW

File: tests/run-build.sh [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, write to the Free Software Foundation, Inc.,
+# 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+
+
+if [ "$1" == "" ]; then
+    BUILD_DIR=$(mktemp -d -p /tmp build_XXXXX)
```
Comment (rewitt1) [inline tests/run-build.sh]: I wouldn't pick the build dir as /tmp. I would let it use whatever it wants, which by default may be /tmp. But mktemp also honors $TMPDIR if the user has it set, and the -p causes it to ignore it.

Also in the case of success, you aren't deleting BUILD_DIR, which I'm guessing wasn't intentional?

File: tests/run-build.sh [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+fi
+# check some artifacts
+STATDIR=`find tmp/buildstats/ -iname quilt-native* -type d`
+for f in `ls $STATDIR`; do
+    echo checking $f
+    if ! grep PASSED $STATDIR/$f | egrep Status; then
+	failure
+    fi
+done
+exit 0
```
Comment (rewitt1) [inline tests/run-build.sh]: Any reason you're explicitly exiting with 0 here? It shouldn't be necessary.

Comment (bavery22) [reply tests/run-build.sh]: removed.

File: tests/run-build.sh (hunk 1 above)
Comment (bavery22) [reply tests/run-build.sh]: fair enough. now building in /workdir/build_XXX in the container.  cleanup on success.

File: tests/runtests.sh [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, write to the Free Software Foundation, Inc.,
+# 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+
+IMAGE=$1
+
+for i in run-*.sh; do
+    echo Running Test $i
+    LOCAL_WDIR=$(mktemp -d -p /tmp wdir_XXXXX)
```
Comment (rewitt1) [inline tests/runtests.sh]: This one has a /tmp too. I was more concerned with it than the one in the container. I commented on the wrong one.

**Comment by rewitt1** [comment]
Made a couple of indentation changes locally and merged.

**Pull request closed without merging**